homeassistant/helpers/floor_registry.py

```diff
@@ -96,7 +97,8 @@ class FloorRegistry:
     def __init__(self, hass: HomeAssistant) -> None:
         """Initialize the floor registry."""
         self.hass = hass
-        self._store = hass.helpers.storage.Store(
+        self._store: Store[dict[str, list[dict[str, Any]]]] = Store(
```

Comment:
Can we use the same type annotation as done below?

https://github.com/home-assistant/core/blob/be25be396e8454a9e43b94596151a2747e869a84/homeassistant/helpers/floor_registry.py#L253